Sources/DistributedActors/Cluster/Reception/_OperationLogClusterReceptionistBehavior.swift

```diff
@@ -95,11 +95,13 @@ public final class _OperationLogClusterReceptionist {
             context.log.debug("Initialized receptionist")
 
             // === listen to cluster events ------------------
-            context.system.cluster.events.subscribe(
-                context.subReceive(Cluster.Event.self) { event in
-                    self.onClusterEvent(context, event: event)
-                }
-            )
+            Task {
+                await context.system.cluster.events.subscribe(
+                    context.subReceive(Cluster.Event.self) { event in
+                        self.onClusterEvent(context, event: event)
+                    }
+                )
```

Comment:
This isn't thread-safe now 😱 

Cannot access `context` from a different task.

Oh man the lack of send hurts real bad here...

Reply:
```swift
let eventStream = context.system.cluster.events

let subReceive = context.subReceive(Cluster.Event.self) { event in
    self.onClusterEvent(context, event: event)
}

Task { eventStream.subscribe(subReceive) } // TODO(send): lack of send was super problematic here, we'd have caused a race condition by accessing `context` from `Task` (!!!)
```

Reply:
😱 will fix

Reply:
FYI @DougGregor some "fun" examples how missing send and therefore forcing people into entering a `Task{}` just to call a void async function is causing all kinds of ugly code and race conditions when integrating with other code.